docs/src/modules/components/MaterialFreeTemplatesCollection.js

```diff
@@ -138,56 +147,60 @@ export default function Templates() {
                 See live preview
               </Button>
             </Box>
-            <Box sx={{ p: 2, pt: 1.5, borderTop: '1px solid', borderColor: 'divider' }}>
-              <Typography
-                component="h3"
-                variant="body1"
-                gutterBottom
-                sx={{ fontWeight: 'semiBold' }}
-              >
-                {layout.title}
-              </Typography>
-              <Typography variant="body2" gutterBottom sx={{ color: 'text.secondary', mb: 2 }}>
-                {layout.description}
-              </Typography>
+            <Box
+              sx={{
+                display: 'flex',
+                justifyContent: 'space-between',
+                p: 2,
+                borderTop: '1px solid',
+                borderColor: 'divider',
+              }}
+            >
               <Box
                 sx={{
                   display: 'flex',
-                  flexDirection: { xs: 'column', sm: 'row' },
+                  alignItems: 'center',
                   gap: 1,
-                  mt: 'auto',
                 }}
               >
-                <Button
-                  component="a"
-                  href={layout.href}
-                  size="small"
-                  fullWidth
-                  variant="outlined"
-                  color="secondary"
-                  startIcon={<Visibility sx={{ mr: 0.5 }} />}
-                  data-ga-event-category="material-ui-template"
-                  data-ga-event-label={layout.title}
-                  data-ga-event-action="preview-img"
-                >
-                  Live preview
-                </Button>
-                <Button
-                  component="a"
-                  href={layout.source}
-                  size="small"
-                  fullWidth
-                  variant="outlined"
-                  color="secondary"
-                  startIcon={<CodeRoundedIcon sx={{ mr: 0.5 }} />}
-                >
-                  Source code
-                </Button>
+                <Tooltip title="Edit in StackBlitz">
+                  <IconButton color="primary" size="small">
+                    <SvgIcon viewBox="0 0 19 28">
+                      <path d="M8.13378 16.1087H0L14.8696 0L10.8662 11.1522L19 11.1522L4.13043 27.2609L8.13378 16.1087Z" />
+                    </SvgIcon>
+                  </IconButton>
+                </Tooltip>
+                <Tooltip title="Edit in CodeSandbox">
```

Comment:
@siriwatknp could you help me to add the functionality to these buttons?

Reply:
Done!